### ISSUE TITLE
Remote development(docker container), python interpreter(virtual environment) code hover doesn't work

### ISSUE THREAD

**Issue by messi-yang** [microsoft/vscode-docs#2741]
Hi, vs code guys.
There is an issue about remote development(docker container).
When I attached vs code for a running python container, and then chose a virtual environment for python interpreter, code hover and go to definition(and something else about language service) doesn't work.
But if I switch to container's default interpreter(python2.7.12), it can work.
I think something wrong about the container's virtual environment interpreter.

Environment:
vs code version: 1.35.0
extensions: Python, GitLens, Visual Studio IntelliCode.
OS: MAC OSX: 10.14.5

Thank you, guys.

**Comment by messi-yang** [comment]
Addition:
When the cursor is in python module or functions, it just shows "Code Analysis is in progress..." and never finished.

**Comment by jsleb333** [comment]
I've encountered a similar issue, where the autocompletion only displays _No suggestions_, hovering with the cursor just shows "Code analysis is in progress..." indefinitely and Go to definition does not work. I've found that this happens only when the selected interpreter is the one provided by conda. One of my colleague has the same issue. This suggests that there is some problem between the Python extension and some Python environments, because it occurs even with Intellicode disabled.

Environment:
VS code Insiders: 1.36.0-insider
Extensions: Python
Interpreter: Python 3.7.1 64-bit ('base':conda)
OS: Ubuntu 18.04.2

In the hope that this can give some more insights.
